api/workspace_helpers.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"encoding/json"
+
+	cloud "github.com/mattermost/mattermost-cloud/model"
+)
+
+func convertInstallationToWorkspace(installation *cloud.InstallationDTO) *Workspace {
```

Comment:
I think we could use some simple test to verify we wont mix up different fields, when we will be changing something here.